### DISCUSSION TITLE
Istio service mesh configurations

### DISCUSSION THREAD

**Discussion by ashishshete133** [linkerd/linkerd2#5846] [Q&A]
I have enabled Istio service mesh but no communication is happening between service A to Service B.

Error:404
cannot find/resolve host http

traffic is reaching till envoy proxy but the service application cannot resolve the request

**Comment by wmorgan** [comment]
The problem is that you installed Istio. Please uninstall it and try again.

**Comment by ashishshete133** [comment]
But that is of no use as I am trying to reach the service through service mesh and not without it.



Sent from Yahoo Mail for iPhone


On Tuesday, March 2, 2021, 2:20 AM, William Morgan <notifications@github.com> wrote:




The problem is that you installed Istio. Please uninstall it and try again.

**Comment by wmorgan** [comment]
I see. After uninstalling Istio, can you please install Linkerd and try again?
